website/src/ethereum-providers/index.js

```diff
@@ -0,0 +1,114 @@
+import frame from './icons/Frame.png'
+import cipher from './icons/Cipher.png'
+import metamask from './icons/Metamask.png'
+import status from './icons/Status.png'
+import wallet from './icons/wallet.svg'
+import fortmatic from './icons/Fortmatic.svg'
+import portis from './icons/Portis.svg'
+
+// See the corresponding prop type, EthereumProviderType, in prop-types.js.
+const PROVIDERS = new Map(
```

Comment:
Same comment as for the other array of objects, if you could condense this I would appreciate it

Reply:
Done in https://github.com/1Hive/apiary/pull/181/commits/9fc22511a86e8d560ecf888b640de76873a24829.